obspy/io/seisan/tests/test_core.py

```diff
@@ -195,13 +194,8 @@ def test_read_obspy(self):
         # 1 - little endian, 32 bit, version 7
         st1 = read(os.path.join(self.path,
                                 '2011-09-06-1311-36S.A1032_001BH_Z'))
-        # raises "UserWarning: Record contains a fractional seconds" - ignore
-        with warnings.catch_warnings(record=True) as w:
-            warnings.simplefilter('always', UserWarning)
-            st2 = read(os.path.join(self.path,
-                                    '2011-09-06-1311-36S.A1032_001BH_Z.mseed'))
-            self.assertEqual(len(w), 1)
-            self.assertEqual(w[0].category, UserWarning)
```

Comment:
why is this no longer necessary?

Reply:
Reading this file triggers the invalid warning that I got rid of in mseed reading (i.e. potentially trialing the wrong endian in the first attempt).

The only thing that changes in here is that the warning-catching was removed, nothing else.